src/test/java/duke/DeadlineTest.java

```diff
@@ -0,0 +1,17 @@
+package duke;
+
+import org.junit.jupiter.api.Test;
+
+import java.time.LocalDate;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class DeadlineTest {
+    @Test
+    public void testStringConversionAndComplete() {
```

Comment:
Consider using this naming format for test cases?
![image](https://user-images.githubusercontent.com/66118590/91692332-ca9d8300-eb9b-11ea-88ea-6345de34d7b2.png)

Reply:
Will do! Thank you!